Source/Scene/ModelAnimation.js

```diff
@@ -230,4 +240,24 @@ Object.defineProperties(ModelAnimation.prototype, {
     },
   },
 });
+/**
+ * A function used to compute the local animation time for a ModelAnimation.
+ * @callback ModelAnimation.AnimationTime
+ *
+ * @param {Number} duration The animation's duration in seconds.
+ * @returns {Number} The fractional part of the return value gives
```

Comment:
This comment seems a bit complicated, but I have to think about how to best rewrite / summarize this. Will come back to it later